docs/toolboxes/modules/ROOT/pages/modeling-analysis-using-json-files.adoc

```diff
@@ -42,36 +42,40 @@ Here is an example of this section for the `fluid` toolbox.
 
 == Parameters
 
-This section of the Model JSON file defines the parameters that may enter expressions used in the subsequent sections.
+This section of the Model JSON file defines the parameters that may enter inside symbolic expressions (as symbols) used in the subsequent sections.
 
 [source,json]
 .Example of a `Parameters` section
 ----
 "Parameters": <1>
     {
         "ubar":"1.0", <2>
-        "chi":"t<2:t", <3>
-        "pIn": <4>
+        "alpha":"2*ubar:ubar", <3>
+        "beta":"{3*alpha,ubar}:alpha:ubar", <4>
+        "chi":"t<2:t", <5>
+        "pIn": <6>
         {
-            "type":"fit", <5>
-            "filename":"$cfgdir/pin.csv", <6>
-            "abscissa":"time", <7>
-            "ordinate":"pressure", <8>
-            "interpolation":"P1", <9>
-            "expr":"10*t+3:t" <10>
+            "type":"fit", <7>
+            "filename":"$cfgdir/pin.csv", <8>
+            "abscissa":"time", <9>
+            "ordinate":"pressure", <10>
+            "interpolation":"P1", <11>
+            "expr":"10*t+3:t" <12>
         }
     }
 ----
 <1> name of the section
 <2> defines a new parameter `ubar` and its associated value
-<3> defines a new parameter `chi` and its associated expression (currently symbols supported are `x`, `y`, `z`, `nx`, `ny`, `nz`, `t`
-<4> defines a new parameter `pIn` and its definition is given in the subsection below 
-<5> the type of parameter is fit
-<6> the filename of a csv file used for the fitting
-<7> column name of csv file used in abscissa
-<8> column name of csv file used in ordinate
-<9> interpolation type of the fit. Possible values are : `P0`, `P1`, `Spline`, `Akima`
-<10> expression used in order to read the fitted value
+<3> defines a new parameter `alpha` and its associated expression. This expression depends on another symbol, here the parameter `ubar`. The symbol defined by this new parameter is also called `alpha`.
+<4> defines a new parameter `beta` and its associated expression. Here the expression is vector of dimension 2. Consequently, symbols generated by this new parameter are `alpha_x` and `alpah_y` (Currently we cannot use a vector as a symbol).
```

Comment:
```suggestion
<4> defines a new parameter `beta` and its associated expression. Here the expression is vector of dimension 2. Consequently, symbols generated by this new parameter are `beta_x` and `beta_y` (Currently we cannot use a vector as a symbol).
```

Reply:
@romainhild I don't understand, my version is not the same : 
`
 <4> defines a new parameter `beta` and its associated expression. Here the expression is vector of dimension 2. Consequently, symbols generated by this new parameter are `beta_0` and `beta_1` (Currently we cannot use a vector as a symbol).
`

Reply:
ah maybe I made the comment on a previous commit, sorry